app/components/articleSearch/index.tsx

```diff
@@ -194,13 +197,23 @@ const SearchContainer: React.FC<Props> = props => {
   const [queryParams, setQueryParams] = React.useState<SearchPageQueryParams>(
     parse(location.search, { ignoreQueryPrefix: true })
   );
+  const [useAutoYearFilter, setUseAutoYearFilter] = React.useState(true);
   const [filter, setFilter] = React.useState(SearchQueryManager.objectifyPaperFilter(queryParams.filter));
   const cancelToken = React.useRef(axios.CancelToken.source());
 
+  React.useEffect(
```

Comment:
if you set useEffect like this, it calls API twice.
![image](https://user-images.githubusercontent.com/9796103/60702011-8fbca280-9f38-11e9-8dcd-f4ae8b663bdd.png)